.travis.yml

```diff
@@ -10,6 +10,8 @@ env:
   matrix:
   - INTEGRATION_SPECS=0
   - INTEGRATION_SPECS=1
+services:
+- redis-server
```

Comment:
Oooh, why do we need redis, as a curiosity?

Reply:
We really only need it when running with `INTEGRATION_SPECS=1`, but I didn't feel like going all `matrix.include` :sweat_smile: 

During runtime, redis is used via `lib/pupcycler/store.rb` to track all of the device state.